test-runner-poc/lib/metadata_ui.js

```diff
@@ -4,8 +4,8 @@
  * Module dependencies.
  */
 var Mocha = require('mocha'),
```

Comment:
If the changes in this file are just formatting, can we remove these changes from this PR?